ehsm_kms_service/test/test_kms_with_rest.py

```diff
@@ -2,157 +2,146 @@
 import json
 import argparse
 import base64
+import time
+import random
+
+appid = 202112101919
+nonce = ''
+nonce = nonce.join(random.choice('0123456789') for i in range(16))
+timestamp = int(time.time())
+
+
+params = {}
+params["appid"] = appid
+params["nonce"] = nonce
+params["timestamp"] = timestamp
+params["sign"] = 'timestamp'
```

Comment:
Hardcode a key, paired with appid, and calculate the HMAC-SHA256 as the signature.

Reply:
it will be fix